src/Temporalio/Client/HttpConnectProxyConfig.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+
+namespace Temporalio.Client
+{
+    /// <summary>
+    /// HTTP connect proxy options for Temporal connections.
+    /// </summary>
+    public class HttpConnectProxyConfig : ICloneable
```

Comment:
In addition to a parameterless constructor, we should have one that accepts `TargetHost` and we should make sure in `ToInteropOptions` that this is non-null (i.e. it is "required" but not required on class construction).

Reply:
That makes sense.

Reply:
Updated